docs/guides/seccomp.md

```diff
@@ -192,49 +265,18 @@ When this operator is
 in the cluster, the seccomp gadget can generate `seccompprofile` resources
 that can be used directly by our pods.
 
-To do this, we need to change the `outputMode` field to `ExternalResource` and
-the `output` field to the namespace and name of the desired resource. You
-can edit the trace file and re-apply it. Our example directory already
-includes a modified version of the trace in `trace-external.yaml`:
-
-```
-apiVersion: gadget.kinvolk.io/v1alpha1
-kind: Trace
-metadata:
-  name: seccomp
-  namespace: gadget
-spec:
-  node: demo-node
-  gadget: seccomp
-  filter:
-    namespace: seccomp-demo
-    podname: hello-python
-  runMode: Manual
-  outputMode: ExternalResource
-  output: seccomp-demo/hello-profile
-```
-
-**Remember that you need to set the right node name**.
-
-We can apply this modified file, which will change the existing resource.
-We'll need to capture the syscalls once again, from the start and then call
-`generate` again to get the profile generated:
-
-```
-$ kubectl apply -f docs/examples/seccomp/trace-external.yaml
-trace.gadget.kinvolk.io/seccomp configured
+To do this, we need to use the `--seccomp-profile` option of the command line.
 
-# Delete the pod and stop the previous trace
+```bash
+# Delete the pod.
 $ kubectl delete -f docs/examples/seccomp/unconfined.yaml
 pod "hello-python" deleted
-$ kubectl annotate -n gadget trace/seccomp gadget.kinvolk.io/operation=stop
-trace.gadget.kinvolk.io/seccomp annotated
 
-# Start a new trace and create the pod again
-$ kubectl annotate -n gadget trace/seccomp gadget.kinvolk.io/operation=start
-trace.gadget.kinvolk.io/seccomp annotated
+# Create the pod and start a new trace again
 $ kubectl apply -f docs/examples/seccomp/unconfined.yaml
 pod/hello-python created
+$ kubectl gadget seccomp-advisor start -n seccomp-demo -m seccomp-profile --seccomp-profile-name seccomp-demo/hello-profile -p hello-python
+TAyR9BXes6GU04rG
```

Comment:
Are you sure?
The example was like this before, there is a dedicated section to catch all syscalls needed to bring up the pod.

Reply:
Yes, otherwise, the "confined" pod will not even be able to start because the seccomp will block the syscalls it needs during the bring up.

BTW, the guide was already starting the gadget before deploying the pod:

https://github.com/kinvolk/inspektor-gadget/blob/9999019ac778d543ec09a7a1aae84ff3a8eeef7f/docs/guides/seccomp.md#L233-L237

Reply:
You are right, I removed my comment when I saw I was wrong, sorry for the inconvenience.